:000000 100644 0000000 253b6f6 A	debian/hmaccalc.install

diff --git a/debian/hmaccalc.install b/debian/hmaccalc.install
new file mode 100644
index 0000000..253b6f6
--- /dev/null
+++ b/debian/hmaccalc.install
@@ -0,0 +1,15 @@
+usr/libexec/libkcapi/md5sum
+usr/libexec/libkcapi/sha1sum
+usr/libexec/libkcapi/sha3sum
+usr/libexec/libkcapi/sha256sum
+usr/libexec/libkcapi/sha1hmac
+usr/libexec/libkcapi/sha224hmac
+usr/libexec/libkcapi/sha224sum
+usr/libexec/libkcapi/sha256hmac
+usr/libexec/libkcapi/sha256sum
+usr/libexec/libkcapi/sha384hmac
+usr/libexec/libkcapi/sha384sum
+usr/libexec/libkcapi/sha512hmac
+usr/libexec/libkcapi/sha512sum
+usr/libexec/libkcapi/sm3hmac
+usr/libexec/libkcapi/sm3sum
