--- a/debian/libkcapi1.symbols
+++ b/debian/libkcapi1.symbols
@@ -18,6 +18,8 @@
  LIBKCAPI_1.3.0@LIBKCAPI_1.3.0 1.2.1
  LIBKCAPI_1.3.1@LIBKCAPI_1.3.1 1.3.1
  LIBKCAPI_1.4.0@LIBKCAPI_1.4.0 1.4.0
+ LIBKCAPI_1.5.0@LIBKCAPI_1.5.0 1.5.1-2gl4
+ LIBKCAPI_1.5.1@LIBKCAPI_1.5.1 1.5.1
  kcapi_aead_authsize@LIBKCAPI_0.12.0 1.0.0
  kcapi_aead_blocksize@LIBKCAPI_0.12.0 1.0.0
  kcapi_aead_ccm_nonce_to_iv@LIBKCAPI_0.12.0 1.0.0
@@ -131,6 +133,10 @@
  kcapi_md_sha256@LIBKCAPI_1.3.1 1.3.1
  kcapi_md_sha384@LIBKCAPI_1.0.0 1.0.0
  kcapi_md_sha384@LIBKCAPI_1.3.1 1.3.1
+ kcapi_md_sha3_224@LIBKCAPI_1.5.0 1.5.1-2gl4
+ kcapi_md_sha3_256@LIBKCAPI_1.5.0 1.5.1-2gl4
+ kcapi_md_sha3_384@LIBKCAPI_1.5.0 1.5.1-2gl4
+ kcapi_md_sha3_512@LIBKCAPI_1.5.0 1.5.1-2gl4
  kcapi_md_sha512@LIBKCAPI_1.0.0 1.0.0
  kcapi_md_sha512@LIBKCAPI_1.3.1 1.3.1
  kcapi_md_sm3@LIBKCAPI_1.4.0 1.4.0
