:100755 100755 87e0e16 7b8f258 M	debian/rules

diff --git a/debian/rules b/debian/rules
index 87e0e16..7b8f258 100755
--- a/debian/rules
+++ b/debian/rules
@@ -23,3 +23,4 @@ kcapi_tools = \
  --enable-kcapi-encapp \
- --enable-kcapi-dgstapp
+ --enable-kcapi-dgstapp \
+ --enable-kcapi-hasher
 
@@ -48,2 +49,20 @@ override_dh_auto_install-arch:
 
+
+execute_before_dh_install:
+	#sleep 300
+	# https://github.com/gardenlinux/gardenlinux/issues/4193
+	# Coreutils md5sum or sha*sum tools do not check for the FIPS state.
+	# The tools provide by libkcapi do so, hence will place thoses files here instead.
+	# Since we install it via the hmaccalc.install file, those files need be in place beforhand.
+	# Hence, we're using before_dh_install step.
+	mv ./debian/tmp/usr/libexec/libkcapi/md5sum ./debian/tmp/usr/bin/md5sum
+	mv ./debian/tmp/usr/libexec/libkcapi/sha1sum ./debian/tmp/usr/bin/sha1sum
+	mv ./debian/tmp/usr/libexec/libkcapi/sha3sum ./debian/tmp/usr/bin/sha3sum
+	mv ./debian/tmp/usr/libexec/libkcapi/sha224sum ./debian/tmp/usr/bin/sha224sum
+	mv ./debian/tmp/usr/libexec/libkcapi/sha256sum ./debian/tmp/usr/bin/sha256sum
+	mv ./debian/tmp/usr/libexec/libkcapi/sha384sum ./debian/tmp/usr/bin/sha384sum
+	mv ./debian/tmp/usr/libexec/libkcapi/sha512sum ./debian/tmp/usr/bin/sha512sum
+	mv ./debian/tmp/usr/libexec/libkcapi/sha512hmac ./debian/tmp/usr/bin/sha512hmac
+
+
 override_dh_install-arch:
@@ -52,2 +71,15 @@ override_dh_install-arch:
 
+execute_after_dh_strip:
+	# generate the hmac of the striped lib
+	./bin/kcapi-hasher -n sha512hmac ./debian/libkcapi1/usr/lib/$(shell uname -m)-linux-gnu/libkcapi.so.1.5.0 > ./debian/libkcapi1-hmac/usr/lib/$(shell uname -m)-linux-gnu/.libkcapi.so.1.5.0.hmac
+	sed -i 's/.\/debian\/libkcapi1//' ./debian/libkcapi1-hmac/usr/lib/$(shell uname -m)-linux-gnu/.libkcapi.so.1.5.0.hmac
+	./bin/kcapi-hasher -n sha512hmac ./debian/libkcapi1/usr/lib/$(shell uname -m)-linux-gnu/libkcapi.so.1 > ./debian/libkcapi1-hmac/usr/lib/$(shell uname -m)-linux-gnu/.libkcapi.so.1.hmac
+	sed -i 's/.\/debian\/libkcapi1//' ./debian/libkcapi1-hmac/usr/lib/$(shell uname -m)-linux-gnu/.libkcapi.so.1.hmac
+	./bin/kcapi-hasher -n sha512hmac ./debian/kcapi-tools/usr/bin/kcapi-hasher > ./debian/kcapi-tools/usr/bin/.kcapi-hasher.hmac
+	sed -u 's/.\/debian\/kcapi-tools\/usr\/bin\//*/' ./debian/kcapi-tools/usr/bin/.kcapi-hasher.hmac
+
+
+
+
+
 override_dh_auto_test-arch:
